using-valtimo/access-control/backend-access-control.md

```diff
@@ -0,0 +1,114 @@
+# Backend-end access control
+This section describes how Policy Based Access Control (PBAC) can be configured in the backend.
+
+## Autodeployment
+PBAC can be configured in the backend using auto-deployment. 
+The deployment will scan for files on the classpath matching either `**/*.role.json` or `**/*.permission.json` for respectively role- and permission configurations.
+
+Every deployment file for PBAC represents a changeset, much like Liquibase. 
+These files contain a `changesetId` that should be unique over all deployment files that use changesets (currently only PBAC).
+
+The contents of a changeset cannot change as long as the `changesetId` does not. A change to an existing changeset can only be made when the `changesetId` also changes.
+Changes made to the deployment files of PBAC will result in a full recreation of existing role- or permission configuration.
+
+## Configuring roles
+The roles should be defined before permissions can be deployed. The file contains only a list of role names next to the mandatory `changesetId`.
+
+`all.role.json`:
+``` json
+{
+    "changesetId": "pbac-roles-v1",
+    "roles": [
+        "ROLE_USER",
+        "ROLE_ADMIN"
+    ]
+}
+```
+
+
+## Configuring permissions
+The example below defines 2 permissions:
+- A user with `ROLE_ADMIN` can `VIEW` any document
+- A user with `ROLE_USER` can `VIEW` documents where:
+  - the name of the document-definition equals `loans`
+  - the `height` of the loan is less than 20000
+
+`document.permission.json`:
+``` json
+{
+    "changesetId": "pbac-documents-v1",
```

Comment:
I wouldn't use this as a changeset ID. This might give the impression we support versioning for access control, which we do not.